README.md

```diff
@@ -113,6 +113,7 @@ Here are the `conf.yaml` configuration options:
 | `github_app_id`              | ID (a number) of the Github app. | No |
 | `github_app_cert_path`       | Path to a certificate which Github provides as an auth mechanism for Github apps. | No |
 | `refresh_interval`           | Time in seconds between checks on repository. Default is 180 | No |
+| `clone_url`                  | The URL of the Github repository. Can be used to [authenticate using SSH](https://help.github.com/en/enterprise/2.16/user/articles/connecting-to-github-with-ssh) instead of HTTPS by specifying the SSH URL of the repository. | No |
```

Comment:
```suggestion
| `clone_url`                  | The URL of the Github repository. Can be used to [authenticate using SSH](https://help.github.com/en/articles/connecting-to-github-with-ssh) instead of HTTPS by specifying the SSH URL of the repository. | No |
```

Reply:
Ok, I've made the change